src/reader/components/ReaderLine.vue

```diff
@@ -54,3 +62,43 @@
     margin: 20px 0;
   }
 </style>
+
+<style lang="scss">
```

Comment:
Not a blocker on merge, but I'll create a card in the backlog for further discussion...depending on how we make the reader and `ReaderLine` "pluggable", we may want to have more segmentation between the display modes and styling.

![](https://github.trello.services/images/mini-trello-icon.png)  [Modularity of readers / reader components](https://trello.com/c/aDn12rrP)